### ISSUE TITLE
Deleting click event when destroing a toast

### ISSUE THREAD

**Issue by k-ivan** [egoist/native-toast#8]
Hi @egoist ).
If the `closeOnClick` option is enabled, I think it would be nice to delete the handler when destroying the toast, as removing the toast itself does not remove the handler.
https://github.com/egoist/native-toast/blob/609f09cd7b03de4fd838a17925e87e4aa7cf227c/src/js/index.js#L71

**Comment by mnahkies** [comment]
I don't think this is necessary. When the destroy method is called, the element is removed from the DOM and the reference to it cleared, meaning that the element should be garbage collected, and the click event listener with it.

See here for a more detailed explanation:
https://stackoverflow.com/questions/12528049/if-a-dom-element-is-removed-are-its-listeners-also-removed-from-memory

Is there some situation where the destroy method does not get called / are you observing a memory leak from this?
